app/commands/git/sync_concept_exercise.rb

```diff
@@ -51,6 +70,20 @@ def find_concepts(slugs)
       end
     end
 
+    memoize
+    def author_usernames_config
+      return [] unless head_git_exercise.authors
+
+      head_git_exercise.authors.map { |a| a["exercism_username"] }
```

Comment:
You could also merge these three lines using `to_a`
```
head_git_exercise.authors.to_a.map { |a| a["exercism_username"] }
```

I see pros and cons for either! :)

Reply:
I like the suggestion!